app/controllers/admin/orders_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Admin::OrdersController < ApplicationController
```

Comment:
Why don't you inherit from `Admin::BaseController`? 

In this way you could remove some duplicate code